.github/scripts/auto-backport.py

```diff
@@ -192,21 +185,24 @@ def main():
         if not backport_labels:
             print(f'no backport label: {pr.number}')
             continue
-        if args.commits and not with_github_keyword_prefix(repo, pr):
+        if not with_github_keyword_prefix(repo, pr):
             continue
         if not repo.private and not scylladbbot_repo.has_in_collaborators(pr.user.login):
             logging.info(f"Sending an invite to {pr.user.login} to become a collaborator to {scylladbbot_repo.full_name} ")
             scylladbbot_repo.add_to_collaborators(pr.user.login)
             comment = f':warning:  @{pr.user.login} you have been added as collaborator to scylladbbot fork '
-            comment += f'Please check your inbox and approve the invitation, once it is done, please add the backport labels again\n'
-            create_pr_comment_and_remove_label(pr, comment)
-            continue
+            comment += f'Please check your inbox and approve the invitation, otherwise you will not be able to edit PR branch when needed\n'
+            pr.add_to_labels("scylladbbot/backport_error")
+            pr.create_issue_comment(comment)
+            is_collaborator = False
         commits = get_pr_commits(repo, pr, stable_branch, start_commit)
+        labels += [label.name for label in pr.labels]
         logging.info(f"Found PR #{pr.number} with commit {commits} and the following labels: {backport_labels}")
-        for backport_label in backport_labels:
-            version = backport_label.replace('backport/', '')
-            backport_base_branch = backport_label.replace('backport/', backport_branch)
-            backport(repo, pr, version, commits, backport_base_branch)
+        if 'scylladbbot/backport_error' not in labels:
```

Comment:
this should not happen, so i'd suggest error out if this label is included in `labels`.